src/math/src/tests/math_test.cairo

```diff
@@ -224,3 +224,241 @@ fn test_rotr_max() {
     assert(BitRotate::rotate_right(0b101_u128, 127) == 0b1010, 'invalid result');
     assert(BitRotate::rotate_right(0b101_u256, 255) == 0b1010, 'invalid result');
 }
+
+#[test]
+fn test_wrapping_math_non_wrapping() {
+    assert(10_u8.wrapping_add(10_u8) == 20_u8, 'invalid result');
```

Comment:
instead of doing `assert(a == b, 'error message')` you can do `assert_eq!(a,b)` 

see [Assert Macros](https://book.cairo-lang.org/ch10-01-unrecoverable-errors-with-panic.html?highlight=assert_eq#assert-and-assert_eq-macros)

Reply:
Sure! I agree with using the macro here, just kept with the usual `assert` to follow the same standard as the other tests. Still, if you don't think it's a problem using it, that's great.

Reply:
Done in [6c6eef1](https://github.com/keep-starknet-strange/alexandria/pull/241/commits/6c6eef1c369c45aa5839c6766dbe0bf7cb123338).

Reply:
> Still, if you don't think it's a problem using it, that's great.

yeah basically `assert!` and `assert_eq!` are relatively new - not worth it to update existing tests, but new ones should defintely use it as they're so much more practical!